.drone.yml

```diff
@@ -0,0 +1,92 @@
+---
+kind: pipeline
+name: default
+
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+  - name: ecs
+    host:
+      path: /etc/profile.d/ecs-credentials-endpoint
+
+clone:
+  depth: 1
+
+steps:
+  - name: cache-restore
+    image: prima/drone-tools:1.15.0
+    volumes:
+      - name: ecs
+        path: /etc/profile.d/ecs-credentials-endpoint
+      - name: docker
+        path: /var/run/docker.sock
+    commands:
+      - . /etc/profile.d/ecs-credentials-endpoint
+      - cache-restore
+
+  - name: elixir-1-8
+    image: elixir:1.8.2
+    environment:
+      MIX_ENV: test
+    commands:
+      - mix local.hex --force
+      - mix deps.get
+      - mix compile --all-warnings --warnings-as-errors
+      - mix test
+      - mix dialyzer --format=dialyzer
+    depends_on:
+      - cache-restore
+
+  - name: elixir-1-9
+    image: elixir:1.9.2
+    environment:
```

Comment:
@VitorTrin we tried to support Elixir from 1.2, as specified in the older `mix.exs`.
However, credo and dialyzer could not seem to work, with the minimum version being elixir 1.6.
We could try and make an effort to support Elixir from 1.6, but it may not be worth the effort to go back to such an older Elixir version that 1.2 is...

Reply:
We are above 1.8, it was only a matter of testing the suported range

Reply:
I agree to bump up the version to 1.6 :+1: